wordpress-mobile/gutenberg-mobile
# Enable VideoPress block on Android

**Related PRs:**
* https://github.com/wordpress-mobile/WordPress-Android/pull/18440

This PR enables the VideoPress block on Android.

**To test:**
1. Open the Android app and connect it with the local Metro server.
2. Open/create a post.
3. Tap ➕ button to open the block inserter.
4. Observe that the VideoPress block is displayed.
5. Tap on the VideoPress block to add it.
6. Add a video.
7. Observe that the video is displayed and can be played.

PR submission checklist:
